components/org.wso2.carbon.identity.api.user.recovery/src/main/java/org/wso2/carbon/identity/recovery/endpoint/Utils/RecoveryUtil.java

```diff
@@ -483,19 +499,45 @@ public static HttpResponse makeCaptchaVerificationHttpRequest(ReCaptchaResponseT
                                                                   Properties properties) {
 
         HttpResponse response = null;
-        String reCaptchaSecretKey = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SECRET_KEY);
         String reCaptchaVerifyUrl = properties.getProperty(CaptchaConstants.RE_CAPTCHA_VERIFY_URL);
+        boolean reCaptchaEnterpriseEnabled =
+                Boolean.valueOf(properties.getProperty(CaptchaConstants.RE_CAPTCHA_ENTERPRISE_ENABLED));
         CloseableHttpClient httpclient = HttpClientBuilder.create().useSystemProperties().build();
-        HttpPost httppost = new HttpPost(reCaptchaVerifyUrl);
-        List<BasicNameValuePair> params = Arrays.asList(new BasicNameValuePair("secret", reCaptchaSecretKey),
-                new BasicNameValuePair("response", reCaptchaResponse.getToken()));
-        httppost.setEntity(new UrlEncodedFormEntity(params, StandardCharsets.UTF_8));
+        HttpPost httppost;
 
-        try {
-            response = httpclient.execute(httppost);
-        } catch (IOException e) {
-            RecoveryUtil.handleBadRequest(String.format("Unable to get the verification response : %s", e.getMessage()),
-                    Constants.STATUS_INTERNAL_SERVER_ERROR_MESSAGE_DEFAULT);
+        if (!reCaptchaEnterpriseEnabled){   // for Recaptcha V2 and V3
```

Comment:
```suggestion
        if (!reCaptchaEnterpriseEnabled) {   
```
Fix formattings. Shall we reverse the condition to get rid of the negation? and also shall we add the comment in a new line? Fix the formattings of comments as well.